source/docs/0.14/services.md

```diff
@@ -364,8 +364,9 @@ module.exports = {
       $dependencyTimeout: 30000 // Default: 0 - no timeout
   },
   dependencies: [
-      "likes",
-      "v2.users"
+      "likes", // shorthand w/o version
+      { name: "users", version: 2 }, // with numeric version
```

Comment:
Please add example as string with version, e.g. `v2.auth`. It works, as well.

Reply:
yes it's updated